.github/workflows/ci.yml

```diff
@@ -43,7 +43,9 @@ jobs:
           - openjpeg
       envs: |
         - macos: py38
-        - windows: py310
+        - windows: py39
+        - linux: base_deps
+        - linux: py39-devdeps
```

Comment:
I still maintain we don't need or want to be running this on PRs. For the majority of contributions it's not a relevant test.

Reply:
I will fix this in my open PR